packages/rule-apple-touch-icons/README.md

```diff
@@ -1,21 +1,21 @@
 # Require an apple touch icon (`@sonarwhal/rule-apple-touch-icons`)
 
-`apple-touch-icons` warns against not using a single `180×180px`
-PNG image for the `apple-touch-icon`.
+`apple-touch-icons` warns against using multiple PNG images when
```

Comment:
How about: Requires that a single 180x180px apple touch icon apple touch icon is provided.